cognite/client/data_classes/transformations/common.py

```diff
@@ -0,0 +1,145 @@
+    """
+
+    def __init__(self, type: str = None):
+        self.type = type
+
+    def __hash__(self):
+        return hash(self.type)
+
+    def __eq__(self, obj):
+        return isinstance(obj, TransformationDestination) and hash(obj) == hash(self)
+
+    @staticmethod
+    def assets():
+        """To be used when the transformation is meant to produce assets."""
+        return TransformationDestination(type="assets")
+
+    @staticmethod
+    def timeseries():
+        """To be used when the transformation is meant to produce time series."""
+        return TransformationDestination(type="timeseries")
+
+    @staticmethod
+    def asset_hierarchy():
+        """To be used when the transformation is meant to produce asset hierarchies."""
+        return TransformationDestination(type="asset_hierarchy")
+
+    @staticmethod
+    def events():
+        """To be used when the transformation is meant to produce events."""
+        return TransformationDestination(type="events")
+
+    @staticmethod
+    def datapoints():
+        """To be used when the transformation is meant to produce numeric data points."""
+        return TransformationDestination(type="datapoints")
+
+    @staticmethod
+    def string_datapoints():
+        """To be used when the transformation is meant to produce string data points."""
+        return TransformationDestination(type="string_datapoints")
+
+    @staticmethod
+    def sequences():
+        """To be used when the transformation is meant to produce sequences."""
+        return TransformationDestination(type="sequences")
+
+    @staticmethod
+    def files():
+        """To be used when the transformation is meant to produce files."""
+        return TransformationDestination(type="files")
+
+    @staticmethod
+    def labels():
+        """To be used when the transformation is meant to produce labels."""
+        return TransformationDestination(type="labels")
+
+    @staticmethod
+    def relationships():
+        """To be used when the transformation is meant to produce relationships."""
+        return TransformationDestination(type="relationships")
+
+    @staticmethod
+    def data_sets():
+        """To be used when the transformation is meant to produce data sets."""
+        return TransformationDestination(type="data_sets")
+
+    @staticmethod
+    def raw(database: str = "", table: str = ""):
+        """To be used when the transformation is meant to produce raw table rows.
+
+        Args:
+            database (str): database name of the target raw table.
+            table (str): name of the target raw table
+
+        Returns:
+            TransformationDestination pointing to the target table
+        """
+        return RawTable(type="raw", database=database, table=table)
+
+
+class RawTable(TransformationDestination):
+    def __init__(self, type: str = None, database: str = None, table: str = None):
+        super().__init__(type=type)
```

Comment:
I think this is safer. Or can should type ever be anything else for an instance of RawTable?
```suggestion
        return RawTable(database=database, table=table)


class RawTable(TransformationDestination):
    def __init__(self, database: str = None, table: str = None):
        super().__init__(type="raw")
```